example/lib/pages/polygon_perf_stress.dart

```diff
@@ -83,10 +84,146 @@ class _PolygonPerfStressPageState extends State<PolygonPerfStressPage> {
             left: 16,
             top: 16,
             right: 16,
-            child: SimplificationToleranceSlider(
-              initialTolerance: _initialSimplificationTolerance,
-              onChangedTolerance: (v) =>
-                  setState(() => simplificationTolerance = v),
+            child: RepaintBoundary(
+              child: Column(
+                children: [
+                  SimplificationToleranceSlider(
+                    initialTolerance: _initialSimplificationTolerance,
+                    onChangedTolerance: (v) =>
+                        setState(() => simplificationTolerance = v),
+                  ),
+                  const SizedBox(height: 12),
+                  Wrap(
+                    alignment: WrapAlignment.center,
+                    spacing: 12,
+                    runSpacing: 12,
+                    children: [
+                      UnconstrainedBox(
+                        child: DecoratedBox(
+                          decoration: BoxDecoration(
+                            color: Theme.of(context).colorScheme.background,
+                            borderRadius: BorderRadius.circular(32),
+                          ),
+                          child: Padding(
+                            padding: const EdgeInsets.only(
+                              left: 16,
+                              right: 16,
+                              top: 4,
+                              bottom: 4,
+                            ),
+                            child: Row(
+                              children: [
+                                const Tooltip(
+                                  message: 'Use Performant Rendering',
+                                  child: Icon(Icons.speed_rounded),
+                                ),
+                                const SizedBox(width: 8),
+                                Switch.adaptive(
+                                  value: usePerformantRendering,
+                                  onChanged: (v) => setState(
+                                    () => usePerformantRendering = v,
+                                  ),
+                                ),
+                              ],
+                            ),
+                          ),
+                        ),
+                      ),
+                      // Not ideal that we have to re-parse the GeoJson every
+                      // time this is changed, but the library gives no easy
+                      // way to change it after
```

Comment:
Agreed. It could be theoretically worked around by just creating a whole new set of `Polygon`s using something like `copyWith`, but this is just easier, and provides a facility to dynamically update the GeoJson in future.
I might change it later when/if I add a `Polygon.copyWith` when the parameters have settled down.